packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -51,22 +52,24 @@ import BaseVisitType from './base-visit-type.component';
 import LocationSelector from './location-selection.component';
 import VisitAttributeTypeFields from './visit-attribute-type.component';
 import styles from './visit-form.scss';
-import { type VisitFormData } from './visit-form.resource';
+import { getDateTimeFormat, type VisitFormData } from './visit-form.resource';
 import VisitDateTimeField from './visit-date-time.component';
 import { useVisits } from '../visits-widget/visit.resource';
 import { useOfflineVisitType } from '../hooks/useOfflineVisitType';
 
 interface StartVisitFormProps extends DefaultWorkspaceProps {
   visitToEdit?: Visit;
-  showVisitEndDateTimeFields: boolean;
+  isCreatingVisit?: boolean;
+  showVisitEndDateTimeFields?: boolean;
 }
 
 const StartVisitForm: React.FC<StartVisitFormProps> = ({
   patientUuid,
   closeWorkspace,
   promptBeforeClosing,
   visitToEdit,
-  showVisitEndDateTimeFields,
+  isCreatingVisit,
```

Comment:
I find this variable name obtuse. When is `isCreatingVisit` meant to be `true`? Since we separate out _editing_ already, i.e., what's the difference between _creating_ a visit and _starting_ a visit?